lib/util.js

```diff
@@ -47,6 +53,19 @@ async function validateWebhook(requestData, secret) {
       signature = requestData.headers["webhook-signature"];
     }
     body = requestData.body;
+    secret = secretOrCrypto;
+    if (customCrypto) {
+      crypto = customCrypto;
+    }
```

Comment:
I think this leaves a raw edge for users of this function. If I misunderstand the documentation and call `validateWebhook(request, crypto)` without passing 3 arguments as is expected when passing a `Request`, `secret` here would be set to `crypto`. Should this be something like:

```suggestion
    if (customCrypto) {
      secret = secretOrCrypto;
      crypto = customCrypto;
    } else {
      crypto = secretOrCrypto;
    }
```

Reply:
Ah I see, I think this is wrong now because in this branch secret must be the second argument. I was trying to do this without inspecting the types, but I think the correct thing to do here is also assert that `secretOrCrypto` is a string.